src/Tabs/Tabs.js

```diff
@@ -140,8 +140,15 @@ const Tabs = React.createClass({
     );
   },
 
+  getTabs() {
```

Comment:
Can you provide the children to this method and use `forEach` instead of `filter`?
```js
React.Children. forEach(children,
```